### DISCUSSION TITLE
osu!lazer tournament client not working for certain features

### DISCUSSION THREAD

**Discussion by Sheppsu** [ppy/osu#13892] [Q&A] [answered]
I first set up the regular tournament client, and then later heard about the laser tournament client so I decided to set it up. I followed the steps documented in the wiki and filled in all the teams, rounds, and currently known bracket info. Showcase and gameplay seem to work fine, but for schedule, mappool, and some of the other features, it's not working, so I am confused as to what I did wrong. I'm not sure what info I need to provide in order to receive help so I'll respond as soon as I can to any replies. Thanks in advance for any help.

**Comment by peppy** [comment, marked as answer]
A good start would be to explain what isn't working, using screenshots or video.

**Reply by Sheppsu** [reply]
The screen for the map pool looks like this. I also tried creating a match using bancho, closing osu, opening the tourney client and setting it up with the lobby, then opening osu!lazer tourney client but it still showed nothing just as below.
![image](https://user-images.githubusercontent.com/49356627/125726949-7d7d4b80-6756-49fe-a2b1-ce477aa45e30.png)

For the schedule tab, it looks like this:
![image](https://user-images.githubusercontent.com/49356627/125727146-155c0253-f401-445d-a4d4-6e3fbffb5cfe.png)

Team Intro and Seeding I'm not worried about, however, they both show a black screen and for Seeding, clicking show first team and show second team do nothing.

**Reply by peppy** [reply]
You need to use the Team/Rounds/Bracket editors to create your tournament. Nothing will work unless you do this first. Which guide did you follow that omitted these steps?

**Reply by Sheppsu** [reply]
The problem is that I actually have filled in the team, round, and bracket info. Maybe I did something wrong?

https://user-images.githubusercontent.com/49356627/125781327-75fc5e70-9f51-429c-bb89-7bb3b15e0fdb.mp4

**Reply by peppy** [reply]
To actually start using the view modes, you need to select a match in the `Bracket` (non edit) screen. Or from the schedule, but that will only show when the matches are within the next few days.

**Reply by Sheppsu** [reply]
Ah it displays now. Thank you!
